### PR TITLE
Remove react-native configuration warnings

### PR DESCRIPTION
**Pull request by jbristol-trav**
## Included in this PR
- Fix for #27
  - Corrected command paths and format

### CODE REVIEW

File: npm-package/react-native.config.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
-               require('node ./node_modules/tealium-react-native/scripts/preunlink.js')]
-};
\ No newline at end of file
+  project: {
+    ios: {},
+    android: {},
+  },
+  commands: [
+    { name: 'postlink', func: () => require('./scripts/postlink.js') },
+    { name: 'preunlink', func: () => require('./scripts/preunlink.js') },
```
Comment (thymikee) [inline npm-package/react-native.config.js]: Please don't add linking hooks as CLI commands. This script only adds tealium maven url for Android project and sounds like a valid manual step to do. The script is also buggy and adds the url even though it's already there. So please remove it entirely and instead improve installation documentation

File: npm-package/react-native.config.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-      android: {}, 
-    },
-    commands: [require('node ./node_modules/tealium-react-native/scripts/postlink.js'),
-               require('node ./node_modules/tealium-react-native/scripts/preunlink.js')]
-};
\ No newline at end of file
+  project: {
+    ios: {},
+    android: {},
+  },
```
Comment (thymikee) [inline npm-package/react-native.config.js]: if there are no custom config settings for project, you can discard it.

**Comment by tamayok** [comment]
@thymikee Thank you for looking into our implementation - we'll take a look at removing the config.js and improving our docs for installation.

**Comment by jbristol-trav** [comment]
This PR was intended to just remove the current warnings, but sounds like more investigation is required from the Tealium side. Should this PR be closed in favor of an issue?

**Comment by thymikee** [comment]
I recommend to remove the file, because since RN 0.60 we have autolinking, where we intentionally removed the link hooks (postlink, prelink, etc). Your configuration has nothing custom besides the these deprecated link hooks, so it's unnecessary to include it. CLI won't complain about it, because no configuration is valid configuration.

**Comment by thymikee** [comment]
Also, `react-native link` now only links assets. We have whole docs on that in the CLI repo: https://github.com/react-native-community/cli/blob/master/docs

**Comment by craigrouse** [comment]
Thanks for the feedback @thymikee. We'll have an update for you very soon.

**Comment by jameskeith** [comment]
react-native.config.js removed in v1.0.7 on advice of @thymikee as autolinking now covers this functionality.

**Pull request closed without merging**